utils/git_test.go

```diff
@@ -16,6 +16,8 @@ import (
 	"testing"
 )
 
+const tempFileName = "myFile.txt"
```

Comment:
I'm not sure it's necessary to create this constant. It will always be created at compile time and it serves only one test. You can keep it as a local variable in the test.